src/run_throttlebot.py

```diff
@@ -683,44 +685,44 @@ def run(sys_config, workload_config, filter_config, default_mr_config, last_comp
+            logging.info('Taking an accounting before decreasing...')
+            logging.info('IMR {} is currently at {}, trying to improve by {}'.format(imr.to_string(),
                                                                               current_imr_alloc,
-                                                                              imr_improvement_proposal)
+                                                                              imr_improvement_proposal))
             # Improving the resource should always be viable at this step
             if check_improve_mr_viability(redis_db, imr, imr_improvement_proposal):
                 new_imr_alloc = imr_improvement_proposal + current_imr_alloc
                 action_taken[imr] = imr_improvement_proposal
                 finalize_mr_provision(redis_db, imr, new_imr_alloc, workload_config)
-                print 'Improvement Calculated: MR {} increase from {} to {}'.format(imr.to_string(),
+                logging.info('Improvement Calculated: MR {} increase from {} to {}'.format(imr.to_string(),
                                                                                     current_imr_alloc,
-                                                                                    new_imr_alloc)
+                                                                                    new_imr_alloc))
                 mimr = imr
                 break
             else:
                 action_taken[imr] = 0
                 new_imr_alloc = imr_improvement_proposal + current_imr_alloc
-                print 'Improvement Calculated: MR {} failed to improve from {} to {}'.format(imr.to_string(),
+                logging.warning('Improvement Calculated: MR {} failed to improve from {} to {}'.format(imr.to_string(),
                                                                                              current_imr_alloc,
-                                                                                             imr_improvement_proposal)
-                print 'This IMR cannot be improved. Printing some debugging before exiting...'
+                                                                                             imr_improvement_proposal))
+                logging.warning('This IMR cannot be improved. Printing some debugging before exiting...')
 
-                print 'Current MR allocation is {}'.format(current_imr_alloc)
-                print 'Proposed (failed) allocation is {}, improved by {}'.format(new_imr_alloc,
-                                                                                  imr_improvement_proposal)
+                logging.warning('Current MR allocation is {}'.format(current_imr_alloc))
+                logging.warning('Proposed (failed) allocation is {}, improved by {}'.format(new_imr_alloc,
+                                                                                  imr_improvement_proposal))
```

Comment:
I think both of these should be at the info level, line 712 contains the warning